Add an example using Tokio runtime

ndelvalle opened the pull request.

Keruspe commented:
Thanks a lot!

Can you drop the crate, moving the dependencies to the dev-dependencies of lapin, and moving the source file to examples/tokio.rs please?

This way it will be at the same place as the others, but most importantly it will be checked by CI.

It would be awesome if you could also add a note saying that for now the reactor is only available for unix.

Thanks again

ndelvalle commented:
> Thanks a lot!
> 
> Can you drop the crate, moving the dependencies to the dev-dependencies of lapin, and moving the source file to examples/tokio.rs please?
> 
> This way it will be at the same place as the others, but most importantly it will be checked by CI.
> 
> It would be awesome if you could also add a note saying that for now the reactor is only available for unix.
> 
> Thanks again

I think I addressed all your comments, let me know if it is good to go 👍🏻

Keruspe commented:
Perfect, thanks!
